framework/base/Security.php

```diff
@@ -30,18 +31,13 @@
 class Security extends Component
 {
     /**
-     * Cipher algorithm for mcrypt module.
-     * AES has 128-bit block size and three key sizes: 128, 192 and 256 bits.
-     * mcrypt offers the Rijndael cipher with block sizes of 128, 192 and 256
-     * bits but only the 128-bit Rijndael is standardized in AES.
-     * So to use AES in mycrypt, specify `'rijndael-128'` cipher and mcrypt
-     * chooses the appropriate AES based on the length of the supplied key.
+     * Block cipher algorithm for encryption and decryption.
      */
-    const MCRYPT_CIPHER = 'rijndael-128';
+    const BLOCK_CIPHER = 'AES';
```

Comment:
Because we want to discourage anyone from changing these. We want it to be not impossible but difficult.